_posts/en/newsletters/2019-11-20-newsletter.md

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Bitcoin Optech Newsletter #73'
+permalink: /en/newsletters/2019/11/20/
+name: 2019-11-20-newsletter
+slug: 2019-11-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of Bitcoin Core 0.19,
+describes some recent updates to Bitcoin services and clients, and
+summarizes recent changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to Bitcoin Core 0.19.0.1:** users are encouraged to upgrade to the
+  latest [release][bitcoin core 0.19.0.1], which contains new features and
+  multiple bug fixes.
+
+- **Upgrade to LND 0.8.1-beta:** this [release][lnd 0.8.1-beta] fixes
+  several minor bugs and adds compatibility with Bitcoin Core 0.19.x.
+
+## News
+
+{% comment %}
+<!-- commit count: git shortlog -no-merges -s v0.18.1..v0.19.0 | awk '{print $1}' | numsum
+     contributor count: wc -l on list in release notes -->
+{% endcomment %}
+
+- **Bitcoin Core 0.19.0 released:** featuring over 1,000 commits from
```

Comment:
Could clarify that the Bitcoin Core maintainers [decided to replace](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2019/bitcoin-core-dev.2019-11-14-19.00.log.html) the initial 0.19.0 release by 0.19.0.1 due to https://github.com/bitcoin/bitcoin/pull/17449.

Reply:
Agreed! Without that important context the version number is really confusing :)